packages/accordions/src/styled/timeline/StyledOppositeContent.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'timeline.opposite.content';
+
+export const StyledOppositeContent = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  flex: 1;
+  margin-right: auto;
+  padding: ${props => `${props.theme.space.base * 5}px ${props.theme.space.base * 4}px`};
+  text-align: ${props => (props.theme.rtl ? 'left' : 'right')};
```

Comment:
Is IE11 support holding us back from using CSS logical properties?

Reply:
I'll take a look at CSS logical properties — looks interesting. Garden components do need to support IE11 for the time being.

Normally, text alignment uses the default alignment and auto adjusts to RTL / LTR accordingly (via `direction` CSS property). In this particular case, timeline items can alternate, so explicit adjustment is needed.